Gemfile

```diff
@@ -6,6 +6,7 @@ group :test do
   gem "simplecov"
   gem "codeclimate-test-reporter"
   gem "byebug", platform: :mri
+  gem "dry-container"
```

Comment:
Thanks for pointing this out @jcmfernandes, but I'm thinking we don't actually need it in the gemspec, since dry-container is a "soft" dependency. I'll probably be removing the gemspec dependency before the next release, so I'm happy for this to stay here :)